src/GovUk.Education.ExploreEducationStatistics.Common/Cache/CacheAttribute.cs

```diff
@@ -45,11 +45,19 @@ public abstract class CacheAttribute : Attribute
         /// </para>
         /// </summary>
         public Type Key { get; }
+        
+        /// <summary>
+        /// Flag signifying that this caching attribute performs caching updates only.
+        /// When an attribute with this flag set is invoked, it will retrieve a fresh
+        /// value rather than a cached item and then set or update it in the cache,
+        /// therefore always caching and then supplying a freshly retrieved value.
+        /// </summary>
+        public bool UpdateOnly { get; } 
```

Comment:
Could also consider calling this `forceUpdate` or `forceRefresh` which might require less explanation?

Reply:
Have chatted with @ntsim about this and decided to go with `forceUpdate`.